tests/disaster-recovery/mdr/test_active_hub_down_and_restore.py

```diff
@@ -0,0 +1,282 @@
+import logging
+import time
+
+import pytest
+
+from ocs_ci.framework.pytest_customization.marks import tier4a, turquoise_squad
+from ocs_ci.framework import config
+from ocs_ci.ocs.acm.acm import AcmAddClusters, validate_cluster_import
+from ocs_ci.ocs.dr.dr_workload import validate_data_integrity
+from ocs_ci.ocs import constants
+from ocs_ci.deployment.deployment import Deployment
+from ocs_ci.ocs.node import get_node_objs
+from ocs_ci.helpers.dr_helpers import (
+    enable_fence,
+    enable_unfence,
+    get_fence_state,
+    failover,
+    relocate,
+    restore_backup,
+    create_backup_schedule,
+    set_current_primary_cluster_context,
+    set_current_secondary_cluster_context,
+    get_current_primary_cluster_name,
+    get_current_secondary_cluster_name,
+    get_passive_acm_index,
+    wait_for_all_resources_creation,
+    wait_for_all_resources_deletion,
+    gracefully_reboot_ocp_nodes,
+    verify_drpolicy_cli,
+)
+from ocs_ci.helpers.dr_helpers_ui import (
+    check_cluster_status_on_acm_console,
+    failover_relocate_ui,
+    verify_failover_relocate_status_ui,
+)
+from ocs_ci.ocs.exceptions import UnexpectedBehaviour
+from ocs_ci.ocs.resources.pod import wait_for_pods_to_be_running
+from ocs_ci.ocs.utils import get_active_acm_index
+from ocs_ci.utility import version
+from ocs_ci.utility.utils import TimeoutSampler
+
+
+logger = logging.getLogger(__name__)
+
+
+@tier4a
+@turquoise_squad
+class TestApplicationFailoverAndRelocateWhenZoneDown:
```

Comment:
file name should be based upon the class/function name IMO for better readibility.